Module#02/api/src/main/java/com/epam/esm/controller/CertificateController.java

```diff
@@ -0,0 +1,143 @@
+package com.epam.esm.controller;
+
+import com.epam.esm.criteria.Criteria;
+import com.epam.esm.criteria.SortOrder;
+import com.epam.esm.exception.CertificateIsExistsException;
+import com.epam.esm.exception.NotFoundException;
+import com.epam.esm.handler.ErrorResponse;
+import com.epam.esm.dto.CertificateDto;
+import com.epam.esm.service.CertificateService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.time.Instant;
+
+import static org.springframework.http.HttpStatus.INTERNAL_SERVER_ERROR;
+import static org.springframework.http.HttpStatus.NOT_FOUND;
+import static org.springframework.http.HttpStatus.OK;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+@RequestMapping(value = "/certificates")
+public class CertificateController {
+    private final CertificateService certificateService;
+
+    @ResponseBody
+    @GetMapping(value = "/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<Object> getCertificateById(
```

Comment:
Remove everything what is possible to remove if it is default config.

	@GetMapping(value = "/{id}", consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
	public CertificateDto getCertificateById(@PathVariable final Long id) {
		return certificateService.getById(id);
	}